main/src/ca/uwaterloo/flix/language/ast/ErasedAst.scala

```diff
@@ -110,10 +106,14 @@ object ErasedAst {
 
   object IntrinsicOperator0 {
 
+    case object Region extends IntrinsicOperator0
+
     case object RecordEmpty extends IntrinsicOperator0
 
     case class GetStaticField(field: Field) extends IntrinsicOperator0
 
+    case class NewObject(name: String, clazz: java.lang.Class[_], methods: List[ErasedAst.JvmMethod]) extends IntrinsicOperator0
```

Comment:
Do we know what name is? Is it the same as the name of the class?

Reply:
Yes it's used in JvmName: https://github.com/flix/flix/blob/25a47b7acb59c6bf3abf23e0e59566ec8c152c9c/main/src/ca/uwaterloo/flix/language/phase/jvm/JvmName.scala#L130